content/pages/kits.md

```diff
@@ -0,0 +1,72 @@
+Title: Ordering, Building, and Operating DIYnamics Kits
+Slug: kits
+
+## The DIYnamics Kits
+
+The DIYnamics Project provides the resources for you to order, build and operate your own rotating tank platform! As detailed in the [About](/pages/about.html) page, these devices are used to model geophysical fluid dynamics. Rotating tanks of water feature fluid behavior analogous to planetary fluid bodies fundamentally impacted by rotation. As part of our Do-It-Yourself approach, our goal is for educators to perform experiments with rotating tank platforms themselves. Hands-on demonstrations provide a valuable opportunity for students to ask questions and draw conclusions in linking experimental results to real world phenomena.
+
+The DIYnamics Kits provide robust rotating tank platforms at various price points. Previously, obtaining your own device required an often custom design and complicated construction process. We developed our family of models to provide cheap, simple and robust solutions for your own device. Each of the below models is well suited for performing fluid dynamics experiments with students. The rotating tank platforms in the hierarchy of models differ in their ability to scale up in terms of speed and rotation rate.
+
+---
+
+## LEGO-Based Tables - Starting at $60
+
+{% img {static}../images/kits/lego/technics_table.PNG 400 Technics Table complete setup %}
+
+The LEGO-Based Tables are 16-inch diameter rotating tank platforms built using LEGOs, a lazy susan and other household items. The tables are perfect for providing an introduction to geophysical fluid dynamics and are safe for audiences of all ages. The LEGO-Based Tables are cheap yet robust enough for a wide range of experiments and available add-ons enable capturing rotating frame footage and changing rotation rate.
+
+### [Learn more about the LEGO-Based Tables here](/pages/lego.html)
+
+---
+
+## DJ Table - Starting at $1000
+
+{% img {static}../images/kits/dj/dj_table.PNG 400 DJ Table complete setup %}
+
+​The DJ Table is a record player based system for demonstrating geophysical fluid dynamics. The system’s tanks come with transparent lids that make them ideal for visualization via phone or camera, and its minimal footprint makes for easy storage and transportation.​ ​
+
+### [Learn more about the DJ Table here](/pages/dj.html)
+
+---
+
+## HT3 Table - Starting at $5000
+
+{% img {static}../images/kits/ht3/ht3_table.PNG 400 HT3 Table complete setup %}
+
+The HT3 Table is a specially designed system including a large rotating tank, lightweight frame, and multi-functional cart that is meant to optimize geophysical fluid dynamics demonstrations while remaining accessible and portable.  This high-end device is ideal for demonstrating simple to advanced geophysical fluid processes, and can be easily stored and moved between locations.
```

Comment:
Our biggest table is the HT3.  The HT3 is an extremely high quality system, nearly research grade, that includes a nearly 3-foot diameter rotating tank on a powerful, reliable motor.  At the same time, its lightweight frame, multi-functional cart, and 100% battery power make it fully portable.  This high-end device is ideal for demonstrating simple to advanced geophysical fluid processes, and its sheer size make it an ideal eye-catcher for public outreach events.